src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,49 +1,53 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
 import java.io.BufferedWriter;
 import java.io.File;
+import java.io.FileReader;
 import java.io.FileWriter;
 import java.io.IOException;
-import java.nio.file.Files;
-import java.util.List;
 
 public class WorkWithFile {
-    protected static String[] readFromFile(String fromFileName) {
+    private static final String BUY = "b";
+    private static final String SUPPLY = "s";
+    private static final String WHITE_SPACE = " ";
+
+    protected static int[] readFromFile(String fromFileName) {
         File file = new File(fromFileName);
-        List<String> statistic = null;
+        int countBuy = 0;
+        int countSupply = 0;
+        String line = null;
         try {
-            statistic = Files.readAllLines(file.toPath());
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            while ((line = bufferedReader.readLine()) != null) {
+                String[] words = line.split(WHITE_SPACE);
+                for (String word : words) {
+                    if (word.startsWith(BUY)) {
+                        countBuy += Integer.parseInt(word.substring(4));
```

Comment:
you used split, you don't need to count symbols to subring from. data can be accessed by index

Reply:
done
but my logic works too =)